OpenConsole.sln

```diff
@@ -2759,6 +2763,43 @@ Global
 		{506FD703-BAA7-4F6E-9361-64F550EC8FCA}.Release|x64.Build.0 = Release|x64
 		{506FD703-BAA7-4F6E-9361-64F550EC8FCA}.Release|x86.ActiveCfg = Release|Win32
 		{506FD703-BAA7-4F6E-9361-64F550EC8FCA}.Release|x86.Build.0 = Release|Win32
+		{416FD703-BAA7-4F6E-9361-64F550EC8FCA}.AuditMode|Any CPU.ActiveCfg = AuditMode|Win32

```

Comment:
Do we want AuditMode and Fuzzing here? Nah...right?

Reply:
> yes the `.0` entry means "actually build this" i think
> otherwise, you do have to have all of those things in the SLN
> the project is not selected to build in the Fuzzing configuration
> It just says, "If it were, Solution configuration Fuzzing|AnyCPU would select Project 4167... Fuzzing|Win32"

Reply:
thanks @DHowett!